owltorrent-common/src/main/java/edu/rice/owltorrent/common/util/AtomicHashableInteger.java

```diff
@@ -0,0 +1,32 @@
+package edu.rice.owltorrent.common.util;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+/**
+ * Atomic Integer where hash and equals only rely on value.
+ *
+ * @author Max Yu
+ */
+public final class AtomicHashableInteger extends AtomicInteger {
```

Comment:
same here

Reply:
This is mainly useful when the Atomic Integer is nested. So for example, if I were to check if two List<AtomicInteger> were equal, they'd rely on if each element in the list's reference were equal. Same with hashing, and same with more complex structures, such as checking if two classes with List<AtomicInteger> fields were equal.

Reply:
sounds good let's remove it